.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -14,15 +14,15 @@ Please fill in as much of the template below as you're able.
 ### Version
 
 <!--
-List the versions of all `tokio-trace` crates you are using. The easiest way to get
+List the versions of all `tracing` crates you are using. The easiest way to get
```

Comment:
I'm just going to assume here that you've searched for `tokio-trace` and `tokio_trace` across all files :p

Reply:
pretty much